operator/api/v1alpha1/securedcluster_types.go

```diff
@@ -65,26 +65,30 @@ type SecuredClusterSpec struct {
 	//+operator-sdk:csv:customresourcedefinitions:type=spec,order=6,displayName="Kubernetes Audit Logs Ingestion Settings"
 	AuditLogs *AuditLogsSpec `json:"auditLogs,omitempty"`
 
+	// Settings relating to process baselines
+	//+operator-sdk:csv:customresourcedefinitions:type=spec,order=7,displayName="Process baslines settings"
```

Comment:
Let's fix the typo and capitalize for consistency with surrounding items. And add a full stop while at it; we're not consistent about this, but maybe we'll get there someday.
```suggestion
	// Settings relating to process baselines.
	//+operator-sdk:csv:customresourcedefinitions:type=spec,order=7,displayName="Process Baselines Settings"
```